lib/altsvc.c

```diff
@@ -189,7 +189,7 @@ static CURLcode altsvc_add(struct altsvcinfo *asi, char *line)
     as = altsvc_create(srchost, dsthost, srcalpn, dstalpn, srcport, dstport);
     if(as) {
       as->expires = expires;
-      as->prio = prio;
```

Comment:
I think the struct should rather hold an `unsigned int prio;`.

Reply:
Yes, thanks!

There may be similar cases, I haven't traced back them all.